pkg/resources/diff_suppressions.go

```diff
@@ -30,18 +30,18 @@ func IgnoreAfterCreation(_, _, _ string, d *schema.ResourceData) bool {
 	return d.Id() != ""
 }
 
-func IgnoreChangeToCurrentSnowflakeValueInShow(keyInShowOutput string) schema.SchemaDiffSuppressFunc {
+func IgnoreChangeToCurrentSnowflakePlainValueInOutput(attrName, keyInOutput string) schema.SchemaDiffSuppressFunc {
```

Comment:
Instead of renmae and changing every invocation, please extract a common part, so that, usually, we can just call a named func, which better explains its purpose.

Reply:
Done.